packages/esm-patient-attachments-app/src/ui-components/pagination/pagination.component.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styles from './pagination.component.scss';
+import Pagination from 'carbon-components-react/es/components/Pagination';
+import { useCurrentPatient } from '@openmrs/esm-framework';
+import chunk from 'lodash-es/chunk';
+
+interface PatientChartPaginationProps {
+  items: Array<unknown>;
+  pageNumber: number;
+  pageSize: number;
+  onPageNumberChange?: any;
+  pageUrl: string;
+  currentPage: Array<unknown>;
+}
+
+/**
+ * @param items The items to be paged.
+ * @param pageNumber The number of the page to be returned as the first result element. Not zero-based!
+ *                   The first page has the number 1.
+ * @param pageSize The number of items per page.
+ * @param onPageNumberChange The function called when page Number is changed
+ * @param pageUrl The url to redirect when see all link is clicked
+ * @param currentPage The currentPage items to be displayed
+ */
+
+const PatientChartPagination: React.FC<PatientChartPaginationProps> = ({
+  items,
+  pageSize,
+  onPageNumberChange,
+  pageNumber,
+  pageUrl,
+  currentPage,
+}) => {
+  const { t } = useTranslation();
+  const [, , patientUuid] = useCurrentPatient();
```

Comment:
Please avoid using `useCurrentPatient`. Usually you get the `patientUuid` as prop anyway directly - so just forward it.

Reply:
Resolved